cmd/cluster-image-registry-operator/main.go

```diff
@@ -52,9 +66,15 @@ func main() {
 	}
 
 	k8sutil.AddToSDKScheme(appsapi.AddToScheme)
+	k8sutil.AddToSDKScheme(authapi.AddToScheme)
+
+	watch("rbac.authorization.k8s.io/v1beta1", "ClusterRole", "", 0)
+	watch("rbac.authorization.k8s.io/v1beta1", "ClusterRoleBinding", "", 0)
```

Comment:
Shouldn't it be authapi.SchemeGroupVersion.String()?

Reply:
NO!

Reply:
?

Reply:
why are we not using rbac.authorization.k8s.io/v1 ?

Reply:
(and same for the resource definitions)

/cc @gabemontero as the samples operator is also creating v1beta1 rbac resources.